src/ejabberd_tls.erl

```diff
@@ -0,0 +1,58 @@
+-module(ejabberd_tls).
```

Comment:
ejabberd_tls is a bit confusing.
maybe `mongoose_fast_tls` would make sense.
And a comment about what this module for and that there is another one for otp ssl.
Do we have a behaviour for this one?

Reply:
tried to keep consistency of the names (ejabberd_zlib, ejabberd_socket)

Reply:
feel free to rename ejabberd_zlib and socket in a new PR.

Reply:
Definitely a good idea for a separate PR, this PR should use consistent names.

Reply:
I think introducing *new* modules with `ejabberd_` prefix is wrong on several levels.

Reply:
let's move this discussion out of the scope of this PR. there are some other changes waiting in queue and depending on this name, i already have a lot of merging work. please do not increase it. we can change the name after the end of the merge of all changes, just create redmine issue for this.

Reply:
@kzemek 

I think it's a lesser evil right now. We won't forget to refactor it for sure.